src/components/ClusterManager.js

```diff
@@ -232,7 +232,7 @@ export default class ClusterManager extends Component {
       }, allClusters)
       this.setState({ clusters })
     } catch (error) {
-      reportError(`Error loading clusters: ${error}`)
+      reportError({ title: 'Error loading clusters', error })
```

Comment:
Why not just `(title, error)` rather than `({title, error})`? It's always going to have both, right?

Reply:
Yeah, I was originally making title optional, but then all of them ended up with titles. I'll simplify.